src/components/ZetkinSpeedDial/ZetkinSpeedDial.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import { ACTIONS } from './constants';
+import ZetkinSpeedDial from '.';
+
+export default {
+  component: ZetkinSpeedDial,
+  title: 'Molecules/ZetkinSpeedDial',
+} as ComponentMeta<typeof ZetkinSpeedDial>;
+
+const Template: ComponentStory<typeof ZetkinSpeedDial> = (args) => (
+  <ZetkinSpeedDial actions={args.actions} />
+);
+
+export const basic = Template.bind({});
+basic.args = {
+  actions: [ACTIONS.CREATE_CAMPAIGN, ACTIONS.CREATE_EVENT, ACTIONS.CREATE_TASK],
```

Comment:
Maybe better to create a new action?

Reply:
explain, i don't understand?

Reply:
You're using the `ACTION.CREATE_CAMPAIGN` but could write a new action? It's just a suggestion, that could show in the code how actions are made.

Reply:
aaah nice idea! Will do :)